proto/agent/v1/connection.proto

```diff
@@ -29,6 +30,19 @@ message UpdateAgentConnection {
     google.protobuf.Duration agent_uptime = 16;
 }
 
+message CloudCommand {
+    MessageType type = 1;
+    google.protobuf.Any payload = 2;
+}
+
+message SendNodeInstances {
+    string claim_id = 1;
+    string mqtt_broker_addr = 2;
```

Comment:
The `mqtt_broker_addr` is no longer neccessery for routing messages inside cloud.
```suggestion
    string mqtt_broker_addr = 2;
```